toplevel/byte/topmain.ml

```diff
@@ -214,7 +195,7 @@ let main () =
   Compenv.parse_arguments ~current argv file_argument program;
   Compenv.readenv ppf Before_link;
   Compmisc.read_clflags_from_env ();
-  if not (prepare ppf) then raise (Compenv.Exit_with_status 2);
+  if not (Toploop.prepare ppf) then raise (Compenv.Exit_with_status 2);
   Compmisc.init_path ();
```

Comment:
Note for later: there is a strange redundancy between `prepare` which calls `Topcommon.set_paths` and this call to `init_path` but this anterior to this PR.